test/algebra.jl

```diff
@@ -1,10 +1,10 @@
 @testset "Algebra" begin
     Mod.@polyvar x y
-    @test 2 .- ((1 .+ (-x)) .* 4) ./ 2 == x.^2 .* (1 ./ x) .* 2
+    #@test 2 .- ((1 .+ (-x)) .* 4) ./ 2 == x.^2 .* (1 ./ x) .* 2
```

Comment:
What is failing here ?

Reply:
There are deprecation warnings related to the new broadcast implementation
```julia
julia> @polyvar x y;
julia> 2 .- ((1 .+ (-x)) .* 4) ./ 2 == x.^2 .* (1 ./ x) .* 2
┌ Warning: broadcast will default to iterating over its arguments in the future. Wrap arguments of
│ type `x::Term{true,Int64}` with `Ref(x)` to ensure they broadcast as "scalar" elements.
│   caller = ip:0x0
└ @ Core :-1
true
```
I am also not sure what the intention of the test was, that everything acts as a scalar? And I am not exactly sure what the broadcast behaviour actually should be, so I punted on this for now...

Reply:
Is there a way to specify that `AbstractTerm` is a scalar so that people don't have to use `Ref` ?

Reply:
Indeed there is, I just had to read the broadcasting docs careful enough. You can do
```julia
Base.broadcastable(t::Term) = Ref(t)
```
to make `Term` not participate in broadcast. I added this now for all types (including `Polynomial`). Is this as intended? Or should polynomials be broadcastable?